Science Use Case - Proposal Prep.ipynb

```diff
@@ -281,8 +322,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "spec_table=Table(hdu_list[1].data)\n",
-    "spec_table"
+    "## Or write it to disk\n",
+    "with open('downloads/{}'.format(spec_tables[0].make_dataset_filename()),'wb') as outfile:\n",
+    "    outfile.write(spec_tables[0].getdataset().read())"
```

Comment:
I'm seeing a couple minor issues with this cell:

1.   It assumes I've already got a `downloads` subdirectory, and errors out if I don't.
2.  Once I have the `downloads` directory, the output file name has a file type of `None` instead of `fits`

Reply:
Yes, fixed.  The second is a known issue (just added to the KNOWN_ISSUES) that I'd seen before but forgotten about with the pyvo make_dataset_filename() method.

Reply:
The fixes look good. 

It looks like changes (from running the notebooks) got committed for 'Science Use Case - Inspecting a Candidate List.ipynb' and 'Science Use Case - Proposal Prep pyvo.ipynb'.  Could you clear/revert those changes prior to the merge?  :)

Reply:
Crap.  Just when I was feeling like I was getting the hang of this.  (Did you notice I rebased?!)